src/main/kotlin/stringcalculator/StringCalculator.kt

```diff
@@ -0,0 +1,52 @@
+package stringcalculator
+
+import java.util.regex.Pattern
+
+class StringCalculator() {
```

Comment:
프로퍼티가 불필요하다면 굳이 주생성자 `()`를 둘 필요가 없을 것 같네요. 😉

```suggestion
class StringCalculator {
```